routes/web.php

```diff
@@ -19,6 +19,10 @@
     Route::get('faset', function () {
         return view('faset');
     });
+
+    Route::get('faset/admin', function () {
```

Comment:
Even ignoring potential benefits for permissions, I think this is the better way to structure URLs